src/families/crypto_org/hw-getAddress.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import Cosmos from "@ledgerhq/hw-app-cosmos";
+import type { Resolver } from "../../hw/getAddress/types";
+import { getEnv } from "../../env";
+
+const CRYPTO_ORG_USE_TESTNET = getEnv("CRYPTO_ORG_USE_TESTNET");
+
+const resolver: Resolver = async (transport, { path, verify }) => {
+  const cosmos = new Cosmos(transport);
+  const cointype = CRYPTO_ORG_USE_TESTNET ? "tcro" : "cro";
```

Comment:
we should use the currency.id to determine if we use testnet or not

as we have a currency `crypto_org_croeseid` we probably can use it to know if we want a testnet or not :)

 https://github.com/LedgerHQ/ledgerjs/blob/master/packages/cryptoassets/src/currencies.js#L2735

basically overall we should try to not use `CRYPTO_ORG_USE_TESTNET ` but use the account.currency / currency contextual info to know if you are on testnet or not. that way it allows to have both testnet accounts and normal accounts at same time & prevent to need an env.

Reply:
do you know how to get the currency in a static way?  
Also I am confused how can I switch currency from mainnet to testnet?  (by default, if I plug my ledger and open the crypto_org app, it uses the currency "crypto_org" , not "crypto_org_testnet")

Reply:
> how can I switch currency from mainnet to testnet?

```sh
ledger-live sync -c crypto_org
ledger-live sync -c crypto_org_croeseid
```

would sync different accounts.


> do you know how to get the currency in a static way?

```js
import { getCryptoCurrencyById } from "../../currencies";

getCryptoCurrencyById("crypto_org_croeseid")
```

Reply:
thanks!

```
import { getCryptoCurrencyById } from "../../currencies";

getCryptoCurrencyById("crypto_org_croeseid")
```

For this part, I mean how to get the currency from user input (-c) in a static way?

Reply:
Currency is part of the options you have in second parameter

`{ path, verify, currency }`

https://github.com/LedgerHQ/ledger-live-common/blob/dcf2db107d8551271e43517139845e4a00cd1c99/src/hw/getAddress/types.js#L26

Reply:
thanks, I removed the env variable and am using the currency in account to do the switch now